packages/react/src/patterns/sections/CardSection/CardSection.js

```diff
@@ -4,20 +4,17 @@
  * This source code is licensed under the Apache-2.0 license found in the
```

Comment:
I'm not seeing any `SCSS` removed from this pull request. Is that accurate?

Reply:
Yeah, @photodow we need some of scss  rules for below.
![image](https://user-images.githubusercontent.com/53789187/75122650-e6b0ec80-566d-11ea-81e8-815d7ff175f6.png)